sycl/doc/extensions/experimental/sycl_ext_oneapi_enqueue_functions.asciidoc

```diff
@@ -633,14 +633,22 @@ a!
 ----
 namespace sycl::ext::oneapi::experimental {
 
-void prefetch(sycl::queue q, void* ptr, size_t numBytes);
+enum class prefetch_type { device, host };
 
-void prefetch(sycl::handler &h, void* ptr, size_t numBytes);
+void prefetch(sycl::queue q, void* ptr, size_t numBytes,
+              prefetch_type type = prefetch_type::device);
+
+void prefetch(sycl::handler &h, void* ptr, size_t numBytes,
+              prefetch_type type = prefetch_type::device);
 
 }
 ----
 !====
-_Effects_: Enqueues a `prefetch` to the `sycl::queue` or `sycl::handler`.
+_Effects_: Enqueues a `prefetch` to the `sycl::queue` or `sycl::handler`. The
+`prefetch_type` enum indicates the destination device the USM data is fetched to:
+i.e. a prefetch to `prefetch_type::device` fetches from the host to the device,
+and `prefetch_type::host` fetches from the device back to host.
```

Comment:
I'd suggest tweaking the wording a bit.  Something like:

> The `type` parameter tells the direction of the prefetch operation.  When the value is `device`, the memory is prefetched to the device associated with the queue.  When the value is `host`, the memory is prefetched to the host, regardless of the device on which it currently resides.

In particular, I think we want to make it clear that the user doesn't need to guess where the memory currently resides when using `host`.